React/tic-tac-toe/src/utils/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
+  try {
+    const serializedState = sessionStorage.getItem('state');
+    if (serializedState === null) {
+      return undefined;
+    }
+    return JSON.parse(serializedState);
+  } catch (err) {
+    return undefined;
+  }
+};
+
+export const saveState = (state) => {
```

Comment:
eslint and prettier error.. check this line!

Reply:
i have no warnings at that line, which is the error?